scripts/exrap_dump.sh

```diff
@@ -1399,7 +1318,7 @@ set -x
 
 export STATUS=YES
 export DUMP_NUMBER=10
-$ushscript_dump/bufr_dump_obs.sh $dumptime 3.00 1 null
+$USHbufr_dump/bufr_dump_obs.sh $dumptime 3.00 1 null
```

Comment:
This change is not required. The original $uscript_dump is defined in JOBSPROC_RAP_DUMP and can be retained. Please revert back to original code:
$ushscript_dump/bufr_dump_obs.sh $dumptime 3.00 1 null